test/metabase/email_test.clj

```diff
@@ -181,25 +185,30 @@
   (let [email-body->regex-boolean (create-email-body->regex-fn regexes)]
     (m/map-vals (fn [emails-for-recipient]
                   (for [email emails-for-recipient]
-                    (-> email
-                        (update :to set)
-                        (update :body (fn [email-body-seq]
-                                        (doall
-                                         (for [{email-type :type :as email-part} email-body-seq]
-                                           (if (string? email-type)
-                                             (email-body->regex-boolean email-part)
-                                             (summarize-attachment email-part)))))))))
+                    (cond-> email
+                      (:to email)  (update :to set)
+                      (:bcc email) (update :bcc set)
+                      true         (update :body (fn [email-body-seq]
+                                                   (doall
+                                                    (for [{email-type :type :as email-part} email-body-seq]
+                                                      (if (string? email-type)
+                                                        (email-body->regex-boolean email-part)
+                                                        (summarize-attachment email-part)))))))))
                 @inbox)))
 
 (defn email-to
   "Creates a default email map for `user-kwd` via `test.users/fetch-user`, as would be returned by `with-fake-inbox`"
-  [user-kwd & [email-map]]
-  (let [{:keys [email]} (test.users/fetch-user user-kwd)]
-    {email [(merge {:from (if-let [from-name (email/email-from-name)]
-                            (str from-name " <" (email/email-from-address) ">")
-                            (email/email-from-address))
-                    :to #{email}}
-                   email-map)]}))
+  ([user-kwd email-map]
+   (email-to user-kwd nil email-map))
+
+  ([user-kwd bcc? & [email-map]]
```

Comment:
`email-map` is already an optional arg here (using the `&` syntax for [variadic functions](https://clojure.org/guides/learn/functions#_variadic_functions)) so it strikes me as a little weird to mix fixed-arity and variadic function signatures in the same function. It's also not super clear when reading the tests that `true` means `:bcc?`.

One option could be to just an options map as a separate optional arg after `email-map` in the variadic part of the function signature. Then it can still be omitted by callers that don't need to use bcc, and it'll default to `nil`, without changing the function signature.
```suggestion
  [user-kwd & [email-map {:keys [bcc?]}]]
```